task/upload.go

```diff
@@ -539,23 +549,26 @@ func uploadTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.Outpu
 	return outputNames, outputLocations, nil
 }
 
-func outputLocations(outURL string, relativePath string, autoMp4s bool) ([]OutputName, []clients.OutputLocation, error) {
+func outputLocations(outURL string, hls string, hlsRelPath string, mp4 string, mp4RelPath string) ([]OutputName, []clients.OutputLocation, error) {
```

Comment:
You didn't introduce the method, so looks like it was already untested but given we have a fairly discrete set of inputs and outputs it should be pretty unit testable - would you mind adding a test in here?

Reply:
Added unit tests.